### ISSUE TITLE
Bot verification rules suggestion

### ISSUE THREAD

**Issue by hoomoongoos** [top-gg/issues#618]
Having a requirement of adding the point of entry of a bot in the description (listing the help command or equivalent) could be very helpful for the overall public experience, since not everyone are as experienced as testers are. This wouldn't take much time from devs either.

**Comment by shivaco** [comment]
Bot rule 8 (listed in [the Discord server, #rules-and-info channel](https://discord.gg/dbl)) states:
> Must have a clear and obvious point of entry (e.g. a working help command)

I doubt forcing developers to specifically list bot commands in the long description is a good idea, as it can:
* take away space from actual bot page design;
* just generally take up unnecessary space when the bot could have a command list available on either the bot's website (if there's one) or within the bot itself via a `help`/`commands` command;
* get annoying for users.

One approach I have in mind for this is, dedicate a specific section/sub-page (some sort of a dropdown) on the bot page specifically for a command list that bot owners could fill out with the bot's up-to-date list of commands.

**Comment by hoomoongoos** [comment]
I wasn't suggesting to have to list all the commands, just the point of entry for the bot, such as the "help" command. It could also be approached by adding a "entry point" section in the page.

<img width="491" alt="Captura de pantalla 2021-03-27 213330" src="https://user-images.githubusercontent.com/60244856/112734111-18b51200-8f44-11eb-97cc-34132d94677d.png">

**Comment by shivaco** [comment]
Apologies for the misunderstanding.

This sounds neat, though possibly fits as an optional field, since some bots may not use commands at all.

**Comment by hoomoongoos** [comment]
Don't worry. I may have explained it wrong.

Having it as an optional field would be great, but, coming back to my original suggestion, I think having it as a **requirement** would improve the user experience a lot (only if the bot needs commands to work).

**Comment by Hopefuls** [comment]
I would suggest renaming it to something more generic or understandable for both bot devs when submitting their bot and bot users wanting to use this bot. Renaming it to things such as `help command` or `command list`.

**Comment by hoomoongoos** [comment]
Yeah, I didn't have any idea on how to name it. "command list" is way better than "entry point"!
